docs/reference/api.yaml

```diff
@@ -620,6 +620,52 @@ paths:
           description: App not found
           schema:
             $ref: "#/definitions/ErrorMessage"
+
+# title: app restart
+# path: /apps/{app}/restart
+# method: POST
+# consume: application/x-www-form-urlencoded
+# produce: application/x-json-stream
+# responses:
+#   200: Ok
+#   401: Unauthorized
+#   404: App not found
+  /1.0/apps/{app}/restart:
+    parameters:
+      - name: app
+        in: path
+        required: true
+        type: string
+        minLength: 1
+        description: App name.
+    post:
+      operationId: AppRestart
+      description: Restart App.
+      tags:
+        - app
+      security:
+        - Bearer: []
+      consumes:
+        - application/x-www-form-urlencoded
+      parameters:
+        - name: process
+          in: process
+          type: number
+          required: true
```

Comment:
The `process` field should be a `string` and its **not** required (a user may request that the whole app be restarted).